### PR TITLE
clean and update packages / apply security fix

### PR DESCRIPTION
**Pull request by chicco785**
## Proposed changes

* Clean packages from unused ones
* Update packages where possible to fix security issues

## Types of changes

What types of changes does your code introduce to the project: _Put
an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation (improve documentation)
- [x] Continuous Integration (functionality that improve automation or testing)
- [ ] Breaking change (fix or feature that would cause existing
      functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after
creating the PR. If you're unsure about any of them, don't hesitate to
ask. We're here to help! This is simply a reminder of what we are going
to look for before merging your code._

- [x] I have read the [CONTRIBUTING][contrib] doc
- [x] I have signed the [Contributor License Agreement][cla]
- [x] I have updated the [RELEASE NOTES][release]
- [x] I have added tests that prove my fix is effective or that my
      feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in
      downstream modules

## Further comments

N/A

[cla]: https://raw.githubusercontent.com/orchestracities/auth-management-ui/master/individual_cla.pdf
    "Martel Open Source Software Individual Contributor License Agreement"
[contrib]: https://github.com/orchestracities/auth-management-ui/blob/master/CONTRIBUTING.md
    "Contributing to auth-management-ui"
[release]: https://github.com/orchestracities/auth-management-ui/blob/master/RELEASE_NOTES.md
    "auth-management-ui Release Notes"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ from the one where you launched the Management UI):
 ```bash
 $ cd configuration-api
 $ npm install
-$ source ../.env
+$ cp ../.env .
```
Comment (chicco785) [inline README.md:184]: not sure why, but with source it was not working, can you try as well?

File: configuration-api/main/config.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 let logger = require('logops');
+const dotenv = require('dotenv');
```
Comment (chicco785) [inline configuration-api/main/config.js:2]: testing locally with debug on, it was clear that when doing `source .env` env variable where not actually loaded on the mac.

File: .env.development [hunk 3]
```diff
@@ -1,12 +1,12 @@
 REACT_APP_URI=http://localhost:3000/
 REACT_APP_ANUBIS_API_URL=http://localhost:8085/
 REACT_APP_CONFIGURATION_API_URL=http://localhost:4000/configuration
-REACT_APP_OIDC_ISSUER=http://keycloak:8080/realms/default
+REACT_APP_OIDC_ISSUER=http://localhost:8080/realms/default
```
Comment (chicco785) [inline .env.development:4]: if you use localhost tenant creation will fail, no?

File: .env.development [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
 REACT_APP_OIDC_CLIENT=configuration
 REACT_APP_OIDC_SCOPE="openid profile email"
 REACT_APP_IMAGE_SIZE=20
 REACT_APP_LOG_LEVEL=debug
-REACT_APP_KEYCLOACK_ADMIN=http://keycloak:8080/admin/realms/default
+REACT_APP_KEYCLOACK_ADMIN=http://localhost:8080/admin/realms/default
```
Comment (chicco785) [inline .env.development:9]: if you use localhost tenant creation will fail, no?

File: configuration-api/test/query.test.js [hunk 5]
```diff
@@ -64,7 +64,7 @@ describe('GraphQL-Query', function () {
           .send(listTenants)
           .expect(200)
           .end((err, res) => {
-            if (err) return done(err);
+            if (err) return done(err);
```
Comment (chicco785) [inline configuration-api/test/query.test.js:67]: some spaces have been added

File: .env.development (hunk 3 above)
Comment (Valecant) [reply .env.development:4]: no, with your configuration i was not able to login on keyckoack

**Review by Valecant — APPROVED** [review]

**Comment by chicco785** [comment]
@Valecant good that you reworked the tests, but please next time make a new PR, so we don't mix different issues resolutions. most probably about the DNS you lost ages about a `c` to much: https://www.keycloak.org/

**Comment by Valecant** [comment]
> @Valecantbene che tu abbia rielaborato i test, ma per favore la prossima volta crea un nuovo PR, quindi non mescoliamo risoluzioni di problemi diversi. molto probabilmente riguardo al DNS che hai perso per secoli `c`: https://www.keycloak.org/

can i close this pull?

**Pull request merged**